packages/dotnet/5.0.201/metadata.json

```diff
@@ -1,5 +1,37 @@
 {
     "language": "dotnet",
     "version": "5.0.201",
-    "aliases": ["cs", "csharp"]
+    "provides": [
+        {
+            "language": "c#.net",
+            "aliases": [
```

Comment:
Alias c# and cs should still be provided

Reply:
Won't they conflict with Mono?

Reply:
shouldn't matter, it will just pick the higher version of the 2

Reply:
Done, I also renamed the test file to test.cs-dotnet to force the tests on dotnet c#